gcp/analytics/src/layout/DashboardLayout/Header/HeaderContent/index.tsx

```diff
@@ -225,28 +280,79 @@ export default function Header() {
           horizontal: 'left'
         }}
         sx={{
-          marginLeft: '-180px'
+          marginLeft: '-180px',
+          display: 'flex'
         }}
       >
-        <Box
-          sx={{
-            width: '767px',
-            height: '269px',
-            color: 'black',
-            background: 'white',
-            padding: '40px',
-            display: 'flex',
-            flexDirection: 'column',
-            justifyContent: 'space-between'
-          }}
-        >
-          {availableTokens.map((token, i) => (
-            <NextLink href={`/markets/${token}/overview`} passHref legacyBehavior key={i}>
-              <Link className="header-link" color="black" underline="none" sx={{ cursor: 'pointer' }}>
-                {token}
-              </Link>
-            </NextLink>
-          ))}
+        <Box sx={{ width: '767px', height: '100%', display: 'flex', justifyContent: 'space-between' }}>
```

Comment:
We're using a lot of hardcoded `px` values in here, is that the convention for this project or do they have some sort of spacing method they use from the theme?